pkg/get/get_test.go

```diff
@@ -4359,3 +4359,56 @@ func Test_DownloadTerrascan(t *testing.T) {
 	}
 
 }
+
+func Test_DownloadGolangciLint(t *testing.T) {
+	tools := MakeTools()
+	name := "golangci-lint"
+
+	tool := getTool(name, tools)
+
+	tests := []test{
+		{
+			os:      "darwin",
+			arch:    arch64bit,
+			version: "v1.48.0",
+			url:     `https://github.com/golangci/golangci-lint/releases/download/v1.48.0/golangci-lint-1.48.0-darwin-amd64.tar.gz`,
+		},
+		{
+			os:      "darwin",
+			arch:    archARM64,
```

Comment:
In case of synthetic tests the described `arm64` behaviour doesn't occur, to mimic a real world scenario, change the unit test's arch to the following const: `archDarwinARM64`.

Reply:
@Jasstkn did you copy this from another example? If you did we may need to fix that one.

What @Shikachuu is saying is that we need the text `arm64` here which is in the `archDarwinARM64` const. MacOS decided to do things differently to Linux which uses the text "aarch64" in the `archARM64 const.

Reply:
Yep, I looked up other test for similar change. I'll fix it right away!

Reply:
fixed